localization/languages/pt-BR.json

```diff
@@ -64,8 +64,8 @@
         "viewTasks": "Ver tarefas",
         "newTask": "Nova tarefa", //"new" is a verb - it is used for a button at the bottom of the task overlay
         "defaultTaskName": "Tarefa %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
-        "taskDeleteWarning": {"unsafeHTML": null}, //missing translation
-        "returnToTask": null, //missing translation
+        "taskDeleteWarning": {"unsafeHTML": "Tarefa excluir!"},
+        "returnToTask": "Retornar tarefa",
         "taskDescriptionTwo": null, //missing translation
```

Comment:
"taskDescriptionTwo" 
description of the second task, how can we define?

Reply:
The original string is this:
```
        "taskDescriptionTwo": "%t and %t", //used to describe a task that has two tabs, %t is replaced with the tab titles
```

Where %t is replaced by two tab names. Example:

<img width="1255" src="https://user-images.githubusercontent.com/10314059/73806141-47938600-478e-11ea-9955-e275d1012da1.png">